docs/examples/aws_tags.rb

```diff
@@ -0,0 +1,23 @@
+require 'chef/provisioning/aws_driver'
+
+with_driver 'aws::us-west-2'
+
+machine 'ref-machine-1' do
+  action :allocate
+end
+
+aws_instance 'ref-machine-1' do
```

Comment:
I'm +1 on using *_options for instances and load balancers over how it is currently.